### PR TITLE
[New Package] funny-manpages

### PR DESCRIPTION
**Pull request by saltedcoffii**

### CODE REVIEW

File: packages/funny_manpages.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  version '2.3'
+  compatibility 'all'
+  source_url 'https://github.com/ltworf/funny-manpages/releases/download/2.3/funny-manpages_2.3.orig.tar.gz'
+  source_sha256 'b3459487abf375d0925494f1c015103e2f3198977b68640bb29bc2c3a8f89c83'
+
+  depends_on 'mandb'
+
+  def self.install
+    system "make", "DESTDIR=#{CREW_DEST_DIR}", "install"
+    system "mkdir #{CREW_DEST_PREFIX} -p"
```
Comment (supechicken) [inline packages/funny_manpages.rb:15]: Rubyize with `FileUtils.mkdir_p CREW_DEST_PREFIX` ?

File: packages/funny_manpages.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+class Funny_manpages < Package
+  description 'funny unix manpages'
+  homepage 'https://github.com/ltworf/funny-manpages'
+  version '2.3'
+  compatibility 'all'
+  source_url 'https://github.com/ltworf/funny-manpages/releases/download/2.3/funny-manpages_2.3.orig.tar.gz'
+  source_sha256 'b3459487abf375d0925494f1c015103e2f3198977b68640bb29bc2c3a8f89c83'
+
+  depends_on 'mandb'
```
Comment (uberhacker) [inline packages/funny_manpages.rb:11]: This is already a core package so no need to list here.

File: packages/funny_manpages.rb [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+  def self.install
+    system "make", "DESTDIR=#{CREW_DEST_DIR}", "install"
+    system "mkdir #{CREW_DEST_PREFIX} -p"
+    system "mv #{CREW_DEST_DIR}/usr/share #{CREW_DEST_PREFIX}/ -v"
```
Comment (supechicken) [inline packages/funny_manpages.rb:16]: Rubyize with `FileUtils.mv './usr/share', CREW_DEST_PREFIX, verbose: true` ?

Comment (uberhacker) [inline packages/funny_manpages.rb:16]: Rubyize with `FileUtils.mv 'usr/share', CREW_DEST_PREFIX`?

File: packages/funny_manpages.rb [hunk 4]
```diff
@@ -8,11 +8,9 @@ class Funny_manpages < Package
...
   source_sha256 'b3459487abf375d0925494f1c015103e2f3198977b68640bb29bc2c3a8f89c83'
 
-  depends_on 'mandb'
-
   def self.install
     system "make", "DESTDIR=#{CREW_DEST_DIR}", "install"
-    system "mkdir #{CREW_DEST_PREFIX} -p"
-    system "mv #{CREW_DEST_DIR}/usr/share #{CREW_DEST_PREFIX}/ -v"
+    FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/share"
+    FileUtils.mv "#{CREW_DEST_DIR}/usr/share/man", "#{CREW_DEST_PREFIX}/share", verbose: true
```
Comment (uberhacker) [inline packages/funny_manpages.rb:14]: Simplify?  `FileUtils.mv 'usr/share/man', "#{CREW_DEST_PREFIX}/share", verbose: true` should work also.

Comment (saltedcoffii) [reply packages/funny_manpages.rb:14]: To me, that reduces readability. Do you agree?

Comment (uberhacker) [reply packages/funny_manpages.rb:14]: All I'm saying is including `#{CREW_DEST_DIR}` is overkill.

Comment (saltedcoffii) [reply packages/funny_manpages.rb]: Fixed

**Pull request merged**